spec/integration/monorepo/after/.bazelversion

```diff
@@ -0,0 +1 @@
+3.7.0
```

Comment:
Can you just clarify why this change?

Reply:
Rules_ios does not work on bazel 4.0.0, because it relies on a method that is deprecated in the newer version.